### PR TITLE
Adds the Military Chaplain

### PR DESCRIPTION
**Pull request by realforest2001**
# Preface - This needs atomisation I know. 😔 
# About the pull request
This is still a work in progress, at least so far as the starting equipment is concerned.

Using the lovely chaplain's office from #3887.

# Explain why it's good for the game
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl: theselfish, forest2001
add: Added the Military Chaplain. Office is located in the maintenance corridor next to the memorial garden.
maptweak: Moved some bits around in this corridor, including creating a mini substation for the Chaplain's Office, Chapel, and that region of maintenance.
add: Slightly re-ordered the job list to put Mess Tech under the Quartermaster properly.
code: Tidied up the Preferences menu to be wider rather than longer.
imageadd: Added spawn landmark icons for Auxiliary Officer and Chaplain.
imageadd: Added Squad and Security HUD icons for Chaplain.
code: Created a proc that checks for allow_gun_usage to reduce code duplication.
/:cl:

### CODE REVIEW

**Comment by morrowwolf** [comment]
To be clear before the next testmerge this needs:
Custom sprites
All of the SS13 sprites need taken out
An office that isn't in a maintenance hallway
Serious standards for both roleplay and gameplay put down on paper with dos and do-nots
Someone to explicitly watch and correct bad behavior in game for a bit after it drops

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 1]
```diff
@@ -411,3 +411,19 @@ Contains most of the procs that are called when a mob is attacked by something
...
 			next_move_slowdown = next_move_slowdown + (SLOWDOWN_AMT_GREENFIRE * resist_modifier)
 			to_chat(src, SPAN_DANGER("The viscous napalm clings to your limbs as you struggle to move through the flames!"))
+
+/mob/living/carbon/human/proc/can_use_weapon()
+	if(job == JOB_CHAPLAIN)//Chaplain is forbidden to use weaponry, regardless of the variable!
+		to_chat(src, SPAN_WARNING("Your holy vow does not allow you to use this!"))
+		return FALSE
+	if(allow_gun_usage)
+		return TRUE
+	if(issynth(src))
```
Comment (BeagleGaming1) [inline code/modules/mob/living/carbon/human/human_defense.dm:421]: Won't this mean that /all/ synths are unable to use guns, even combat synths?

**Comment by realforest2001** [comment]
> > add: Slightly re-ordered the job list to put Mess Tech under the Quartermaster properly.
> > code: Tidied up the Preferences menu to be wider rather than longer.
> > code: Created a proc that checks for allow_gun_usage to reduce code duplication.
> 
> Why all of this in the PR?

Reordered job list isn't necessary in this PR.
Tidied up the preferences menu is necessary because otherwise it breaks the menu by adding another job.
Proc to check for allow_gun_usage is because chaplains can't use guns, and the message on trying to use it just talks about synth programming.

**Comment by BeagleGaming1** [comment]
> Proc to check for allow_gun_usage is because chaplains can't use guns, and the message on trying to use it just talks about synth programming.

No, it does not. Humans that use guns get a generic "You cannot use firearms" text. It checks if your species is synthetic, and if it is, it only then says your programming restricts it.

**Comment by realforest2001** [comment]
Closed pending overhaul

**Pull request closed without merging**